### ISSUE TITLE
NTR: C4-OH: 3-hydroxybutyryl-carnitine high in blood (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8322]
New term request C4-OH: 3-hydroxybutyryl-carnitine high in blood (elthat_130306195238): C4OH

**Comment by pnrobinson** [comment]
@MickeySegal
What does the C4-OH mean here?
See
[(R)-3-hydroxybutyrylcarnitine is an O-hydroxybutyryl-L-carnitine in which the acyl group is specified as 3-hydroxybutyryl.](https://pubchem.ncbi.nlm.nih.gov/compound/Hydroxybutyrylcarnitine)

**Comment by MickeySegal** [comment]
As in https://link.springer.com/article/10.1007/s10545-009-9028-3, "3-hydroxybutyrylcarnitine (C4OH = 2.19 μmol/L; upper cut-off = 0.55 μmol/L)".

There is a whole system of such abbreviations.  I'm not finding the one we used but https://til.dlslab.com/physicians/test-directory-view-test/?test=54607 is similar, as is https://europepmc.org/articles/PMC8790479/table/T1/

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000237
- label: Elevated circulating 3-hydroxybutyryl-carnitine concentration
- parents: [Elevated circulating C4 acylcarnitine concentration (HP:0035016)](https://hpo.jax.org/app/browse/term/HP:0035016)
- pmid: PMID:35095998
- def: Concentration of 3-hydroxybutyryl-carnitine in the blood circulation above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8322
- synonyms:
